docker/Dockerfile

```diff
@@ -24,6 +24,7 @@ FROM node:${NODE_VERSION}
 ARG GITHUB_ACCOUNT=telefonicaid
 ARG GITHUB_REPOSITORY=iotagent-ul
 ARG DOWNLOAD=latest
+ARG SOURCE_BRANCH=master
 
 # Copying Build time arguments to environment variables so they are persisted at run time and can be 
 # inspected within a running container.
```

Comment:
Looking to "twin PR" at https://github.com/telefonicaid/iotagent-json/pull/390/files#diff-ebacf6f6ae4ee68078bb16454b23247dR18 there is a comment block there that maybe could be added also to this Dockerfile in the proper line (just before the commen block for RUN):

```
# IMPORTANT: For a Production Environment Use Docker Secrets to define
#  these values and add _FILE to the name of the variable.

# ENV IOTA_AUTH_USER "iotagent"
# ENV IOTA_AUTH_PASSWORD "iotagent"
```

Reply:
Fixed e23d56e